codestodians/food-basket-front-end
# Fix Box Component props handling

#### What does this PR do?
- Fixes the bug that prevents the Box Component from accepting props wherever it is being rendered

#### Description of Task to be completed?
- Fix Box Component `style` prop bug.
- Convert Box Component from class to functional component.
- Remove runtime error-causing `node-fetch` package.
- Update test script to handle `npm debug` error.
- Deprecate `App.spec.js` due to the insistence for the `fetch` option in the `App` config.
- Update the `NoMatch` component to show the non-existent path supplied in the url.

#### How should this be manually tested?
- Freely pass predefined props to the Box component anywhere it is rendered
- Check the browser console to confirm that errors have disappeared

#### Any background context you want to provide?
- This bug was discovered in a bid to provide a temporarily more descriptive message for non-existent routes a user navigates to

#### What are the relevant pivotal tracker stories?
- [#153633977](https://www.pivotaltracker.com/story/show/153633977)

#### Screenshots (if appropriate)
<img width="1280" alt="screen shot 2017-12-13 at 3 22 46 pm" src="https://user-images.githubusercontent.com/22825700/33943943-e09a4f2e-e01a-11e7-9908-d472f9799e99.png">
